Feature/donde reciclar visual

nico-rodriguez opened the pull request.

Commit:
Added logos

Commit:
Added styles

Commit:
Changed component name

Commit:
Refactored

Separated styles from logic.

Commit:
Adjusted flex value

Commit:
Added component

Represents the view "Dónde reciclar". It has map and box with info.

Commit:
Centered text

Added text-align: center.

Commit:
Fixed typo

Commit:
Changed margin and padding values

Changed px values to % values to make it responsive.

Commit:
Added components

Used to center text inside a component.

Commit:
Refactored

Moved some styles to coresponding sub component.

Commit:
Changed to BoxLogo

Commit:
Merged master

Resolved conflict in App.js. Moved map components to map/component.

Commit:
Moved BodyWrapper to styles folder

Commit:
Moved MapComponent to styles folder

Commit:
Fixed layout for mobile

Commit:
Fix scrolling for mobile

Scroll the page with one finger. Move over map with two finger.

Commit:
Changed map center to Rocha

Commit:
Enlarged InfoComponent on mobile

Commit:
Fixed fonts and layout of map component

Tried to make it look like figma.

alexisartus96 reviewed (commented):
![image](https://user-images.githubusercontent.com/29693380/47258359-8229c000-d470-11e8-94ab-265a8cb2602d.png)

![image](https://user-images.githubusercontent.com/29693380/47258374-b8673f80-d470-11e8-9fd8-99ff38c1220a.png)


- Add space between icons and text on desktop view
- Fix tablet view, try by changing font-size
- Load the page on your phone and try to do zoom out on map because on mine didn't work

Commit:
Merge branch 'master' of https://github.com/Pis-moove-it/reciclando-web into feature/donde-reciclar-visual

# Conflicts:
#	package.json
#	src/app/App.js

Commit:
Fixed code review comments

Spacing between icons and text (laptop).
Font size (tablet).
Using media queries.

Commit:
Merge branch 'feature/donde-reciclar-visual' of https://github.com/Pis-moove-it/reciclando-web into feature/donde-reciclar-visual

Commit:
Added zoom control buttons
